builds/e2e/connectivity.yaml

```diff
@@ -218,6 +218,7 @@ jobs:
           customEdgeHub.image: '$(customEdgeHub.image)'
           test.runtimeLogLevel: '$(test.runtimeLogLevel)'
           testInfo.testName: '$(testInfo.testName)'
+          topology: 'SingleNode'
```

Comment:
We need to inject a `topology` var from yaml->script only for single node connectivity since it shares a deployment file across single-node/nested. For nested connectivity and all longhaul pipelines, deployment files are not shared so the value can be hardcoded.